src/main/java/uk/gov/hmcts/reform/sscs/util/OtherPartyDataUtil.java

```diff
@@ -20,6 +21,15 @@
 
 public class OtherPartyDataUtil {
 
+    public static final List<Benefit> SSCS5_BENEFIT = List.of(TAX_CREDIT,
```

Comment:
Best to get them from the sscsType == SSCS5 on the Benefit class, rather than listing them here